### PR TITLE
Fix CORS problem

### PR DESCRIPTION
**Pull request by HexRabbit**
In order to not violate CORS regulation on web browser,"Access-Control-Allow-Origin: *" header needs to be set while responding to client's POST request.
And before POST request, there's an another OPTION request called preflight request [1], the same header above also needs to be set while responding to this kind of request.

[1] https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -24,27 +27,29 @@ def do_GET(self):
...
+        self.send_header('Access-Control-Allow-Origin', '*')
+        self.end_headers()
         self.wfile.write(json.dumps(response))
 
     def do_POST(self):
 
         request_path = self.path
 
         request_headers = self.headers
-        content_length = request_headers.get("Content-length")
```
Comment (jserv) [inline server.py]: Why would you change the way to specify string literals? You have to minimize the functional changes. Once you wish to introduce syntax tweaks, you can always send another pull request(s).

Comment (HexRabbit) [reply server.py]: Got it. I'll remove the style tweak on string literals and send pull request again.

**Comment by HexRabbit** [comment]
Updated.
